test/e2e/run-api-specs-multichain.ts

```diff
@@ -131,10 +174,54 @@ async function main() {
         ],
       });
 
+      const testCoverageResultsCaip27 = await testCoverage({
+        openrpcDocument: MetaMaskOpenRPCDocument as OpenrpcDocument,
+        transport: createCaip27DriverTransport(driver, `eip155:${chainId}`),
+        reporters: [
+          'console-streaming',
+          new HtmlReporter({
+            autoOpen: !process.env.CI,
+            destination: `${process.cwd()}/html-report-caip27`,
+          }),
+        ],
+        skip: [
+          'eth_coinbase',
+          // these 2 methods below are not supported by MetaMask extension yet and
+          // don't get passed through. See here: https://github.com/MetaMask/metamask-extension/issues/24225
```

Comment:
nit: just noting that there are more than 2 methods below. Maybe just move this comment/reorder the array so its a bit clearer